docs/API.md

```diff
@@ -728,6 +728,18 @@ const Counter = componentFromStream(props$ => {
 })
 ```
 
+### `mapPropsStreamWithConfig()`
```

Comment:
This is WIP, I had a couple of questions about whether you wanted to depreciate setObservableConfig.